### PR TITLE
feature/pet-profile

### PR DESCRIPTION
**Pull request by AustinKelsay**

### CODE REVIEW

**Review by KirstenS13 — COMMENTED** [review]
Should the customer id still be hard-coded to 18 on line 138 in src/api/index.js?

**Comment by AustinKelsay** [comment]
> Should the customer id still be hard-coded to 18 on line 138 in src/api/index.js?

Oh no good catch, I'll change it right now!

**Review by KirstenS13 — APPROVED** [review]
Looks good!

File: src/api/index.js [hunk 1]
```diff
@@ -119,6 +133,7 @@ const getProfileData = authState => {
 const getCustomerInfo = id => dispatch => {
   dispatch({ type: GET_CUSTOMER_INFO_START });
 
+  // I hard coded the id here and I should be changing it back and deleting this comment before pushing
```
Comment (taylorpalmer) [inline src/api/index.js:136]: Did you do this? Do we still need this comment?

**Review by SRattacasa — APPROVED** [review]

**Review by taylorpalmer — APPROVED** [review]

**Pull request merged**